applications/OptimizationApplication/tests/top_opt_test/optimization_parameters.json

```diff
@@ -1,114 +1,161 @@
-            {                    
-                "name"       : "Structure",
-                "type"     : "mdpa",
-                "settings" : {
-                    "domain_size"    : 3,
-                    "input_filename" : "Structure"
+    "optimization_settings": {
+        "model_parts": [
+            {
+                "name": "Structure",
+                "type": "mdpa",
+                "settings": {
+                    "domain_size": 3,
+                    "input_filename": "Structure"
                 }
-            }],            
-            "analyses" : [        
-                            {
-                            "name"       : "Structure_static",
-                            "type"       : "StructuralMechanicsAnalysis",                            
-                            "settings"              : {
-                                "model_parts"       : ["Structure"],
-                                "input_filename" : "primal_parameters.json"
-                                }
-                            }                            
-                    ],
-            "responses" : [ 
-                    {            
-                        "name"                : "mass_Structure",
-                        "type"                : "mass",
-                        "settings":{
-                            "evaluated_objects": ["Structure"],
-                            "control_types": ["material"],
-                            "controlled_objects": ["Structure"],
-                            "gradient_settings":{
-                                "step_size" : 1e-8,
-                                "gradient_mode": "finite_differencing"
-                            }              
-                        }
-                    },                 
-                    {            
-                        "name"                : "strain_energy",
-                        "type"                : "strain_energy",
-                        "settings":{
-                            "analysis_name"   : "Structure_static",
-                            "evaluated_objects": ["Structure"],
-                            "control_types": ["material"],
-                            "controlled_objects": ["Structure"],
-                            "gradient_settings":{
-                                "step_size" : 1e-8,
-                                "gradient_mode": "semi_analytic"
-                            }                       
+            }
+        ],
+        "analyses": [
+            {
+                "name": "Structure_static",
+                "execution_policy_type": "KratosMultiphysics.OptimizationApplication.execution_policies.stepping_analysis_execution_policy.SteppingAnalysisExecutionPolicy",
+                "execution_policy_settings": {
+                    "analysis_type": "KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis.StructuralMechanicsAnalysis",
```

Comment:
@sunethwarna The same point here, can't we simply say StructuralMechanicsAnalysis ?! 
Is there any specific reason to not use only the standard names of the applications or solvers ?!

Reply:
Would it be ok, if I shorten it but still keeping the flexibility?

Reply:
Sure, it would be great